eng/common/scripts/Verify-RestApiSpecLocation.ps1

```diff
@@ -259,8 +264,8 @@ try{
   }
   elseif ($Language -eq "java" -or $Language -eq "javascript" -or $Language -eq "python" -or $Language -eq "go") {
     # for these languages we ignore the validation because they always use the latest spec from main branch to release SDK
-    # mgmt plane packages: azure-core-management|azure-resourcemanager|azure-resourcemanager-advisor (java), azure-mgmt-devcenter (python), arm-advisor (js), armaad (go)
-    if($PackageName -match "^(arm|azure-mgmt|azure-resourcemanager|azure-core-management)[-a-z]*$") {
+    # mgmt plane packages: azure-core-management|azure-resourcemanager|azure-resourcemanager-advisor (java), azure-mgmt-devcenter (python), azure-arm-advisor (js), armaad (go)
+    if ($PackageName -match "^(arm|azure-mgmt|azure-resourcemanager|azure-core-management|azure-arm|azure-rest-arm)[-a-z]*$") {
```

Comment:
@qiaozha , can you confirm the name prefix of JS arm packages?

Reply:
You should be able to use the pkgInfo.SdkType for this check.